nodejs-server/app.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const MongoClient = require('mongodb').MongoClient;
+const morgan = require('morgan');
+const app = express();
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const url = 'mongodb://127.0.0.1:27017';
+
+// Connect to the database
+MongoClient.connect(url, {
```

Comment:
Delete `nodejs-server/` directory

All the server related stuff should be inside `packages/server`